### DISCUSSION TITLE
Change filter in data-ajax-cart-bind-state

### DISCUSSION THREAD

**Discussion by mircomartin** [EvgeniyMukhamedjanov/liquid-ajax-cart#46] [Q&A]
Hello i have other question! really sorry if i bother you, 
I would like to change the filter {{ cart.total_price | money_with_currency }}, for | money.
Thanks!

**Comment by EvgeniyMukhamedjanov** [comment]
You can change it. The **| money** filter doesn't work?

**Comment by mircomartin** [comment]
Not working, if you can see the screen it's plain text <p class="flex font-bold justify-between uppercase">
![image](https://user-images.githubusercontent.com/64798218/193409436-0205af9a-82f1-4053-af06-a1d055b8ad17.png)
```
Subtotal:
<span data-ajax-cart-bind-state="cart.total_price | money">
	{{ cart.total_price | money }}
</span>
</p>
```

**Reply by EvgeniyMukhamedjanov** [reply]
@mircomartin 
Sorry for the late answer.
The `data-ajax-cart-bind-state` attribute supports only the | money_with_currency
I would suggest you not to use the `data-ajax-cart-bind-state` attribute here, instead, just wrap the subtotal with the `data-ajax-cart-section` attribute if the subtotal is inside of a shopify section:
```liquid
<div data-ajax-cart-section>
<p>
Subtotal:
<span data-ajax-cart-bind-state="cart.total_price | money">
	{{ cart.total_price | money }}
</span>
</p>
</div>
```
